.github/workflows/pgspot.yaml

```
@@ -0,0 +1,49 @@
+name: pgspot
+on:
+  pull_request:
+
```

It might also be useful to run the workflow as well when changes are made to the `prerelease_test` branch (e.g., like the `linux-build-and-test.yaml` workflow)

```suggestion
  push:
    branches:
      - prerelease_test
```